### PR TITLE
Instructor: help page: make style of sections consistent #8237

### LINKED ISSUE #8237: Instructor: help page: make style of sections consistent
**Description of feature/enhancement**
Noticed stylistic inconsistencies in `instructorHelp.jsp` while working on #8224. For example:

- Paragraph spacing using `<br>` tags not consistent -- should perhaps use `<p>`?
- Inconsistent use of heading styles and numbering (compare FAQ and Sections)
- Missing example divs around some examples

Some examples of the inconsistencies:
screen shot 2017-12-22 at 1 57 11 am

**Justification**
Stylistic consistency will help readers identify logical sections in the text and more easily find the information they are looking for.

### PR DESCRIPTION
**Pull request by nidhi98gupta**
Fixes #8237 
I have made the headings consistent.
Wrapped all the examples with example division.
Gave appropriate spacing wherever required.
Used numbering wherever instructions/steps to achieve something is given, and bullet points for FAQs/questions.
I have removed extra `<br>` 
I  haven't used `<p>` tags yet because I wasn't sure whether it is required or not and where to change it. I can do that if required.
If there are any more changes that I missed, please tell me, I'll fix it.

### CODE REVIEW

**Comment by nidhi98gupta** [comment]
Ready for review.

**Comment by nidhi98gupta** [comment]
@damithc Sir, a reminder.

**Comment by xpdavid** [comment]
Hi @nidhi98gupta Can you give us the screenshots of your changes?

**Comment by nidhi98gupta** [comment]
Hi @xpdavid Here is the link to my staging server (instructorHelpPage) [https://teammates-nidhi.appspot.com/instructorHelp.jsp#gs](url)
Also attaching some screenshots,
screenshot from 2018-01-09 21-45-35
screenshot from 2018-01-09 21-45-20
screenshot from 2018-01-09 21-44-40

File: src/main/webapp/partials/instructorHelpSessions.jsp [hunk 1]
```diff
@@ -697,7 +687,7 @@
       <br>
       <br>
       <p>
-        Here, we set up a particular question after adding it to the session.
+        Here, we set up a   particular question after adding it to the session.
```
Comment (xpdavid) [inline src/main/webapp/partials/instructorHelpSessions.jsp]: Any reason for this?

File: src/main/webapp/partials/instructorHelpSections.jsp [hunk 2]
```diff
@@ -2,10 +2,11 @@
 <h4 class="text-color-primary" id="sections">Sections</h4>
 <div id="contentHolder">
   <br>
-  <ol style="list-style-type: none;">
+  <div class="bulletPointsIndent">
```
Comment (xpdavid) [inline src/main/webapp/partials/instructorHelpSections.jsp]: As discussed

File: src/main/webapp/partials/instructorHelpArchiving.jsp [hunk 3]
```diff
@@ -2,102 +2,113 @@
...
-                      Instructors
-                      <span class="caret"></span>
-                    </button>
-                  </div>
-                  <div class="dropdown courses-table-dropdown">
-                    <button class="btn btn-primary btn-xs" type="button">
-                      Sessions
-                      <span class="caret"></span>
-                    </button>
+  <div class="bulletPointsIndent">
```
Comment (xpdavid) [inline src/main/webapp/partials/instructorHelpArchiving.jsp]: `bulletPointsIndent` the class is not defined in `teammatesCommon.css` (although I don't know why). You should not use it or even delete all.

File: src/main/webapp/partials/instructorHelpSearch.jsp [hunk 4]
```diff
@@ -6,116 +6,120 @@
...
+                          <a class="course-student-delete-link btn btn-default btn-xs"
+                              title="Delete the student and the corresponding submissions from the course" href="" data-toggle="tooltip" data-placement="top">Delete</a>
+                          <a class="btn btn-default btn-xs" href="" title="View all data about this student" target="_blank" rel="noopener noreferrer"
+                              data-toggle="tooltip" data-placement="top">All Records</a>
+                        </td>
+                      </tr>
+                    </tbody>
+                  </table>
+                </div>
               </div>
```
Comment (xpdavid) [inline src/main/webapp/partials/instructorHelpSearch.jsp]: You forget to add example frame to this table
screen shot 2018-01-10 at 8 19 33 pm

File: src/main/webapp/partials/instructorHelpArchiving.jsp (hunk 3 above)
Comment (nidhi98gupta) [reply src/main/webapp/partials/instructorHelpArchiving.jsp]: I have deleted all.

File: src/main/webapp/partials/instructorHelpSessions.jsp (hunk 1 above)
Comment (nidhi98gupta) [reply src/main/webapp/partials/instructorHelpSessions.jsp]: My mistake. Corrected it :)

File: src/main/webapp/partials/instructorHelpSessions.jsp [hunk 5]
```diff
@@ -706,7 +696,7 @@
           <span class="text-muted"> - the question</span>
         </li>
         <li>
-          <b class="feedback-path-title">Feedback Path</b>
```
[the remark this answers is not in the document]
Comment (nidhi98gupta) [reply src/main/webapp/partials/instructorHelpSessions.jsp]: @xpdavid feedback-path-title is used in many places. I felt it got highlighted here by mistake.

File: src/main/webapp/partials/instructorHelpSessions.jsp [hunk 6]
```diff
@@ -2088,10 +2079,11 @@
 
       <div class="row">
         <div class="col-sm-12">
-            <h4 id="fbEssay">Essay Question</h4>
+            <b id="fbEssay">Essay Question</b>
```
[the remark this answers is not in the document]
Comment (nidhi98gupta) [reply src/main/webapp/partials/instructorHelpSessions.jsp]: Size of sub-title was bigger than section title previously. I can add line breaks if you say.

**Comment by nidhi98gupta** [comment]
@xpdavid ready for review :)

**Comment by xpdavid** [comment]
@nidhi98gupta Please update your staging server also :)

**Comment by nidhi98gupta** [comment]
@xpdavid Updated staging server, Please have a look :)

**Comment by xpdavid** [comment]
@nidhi98gupta 

You need a line break here :)
screen shot 2018-01-13 at 10 32 11 am

You forget to add a line break :P
screen shot 2018-01-13 at 10 31 53 am

**Comment by nidhi98gupta** [comment]
@xpdavid Added both the line breaks and deployed the changes :) Please have a look.

**Pull request merged**